integration-tests/test/test_internal.py

```diff
@@ -126,11 +129,12 @@ def test_parse_show_block_output() -> None:
     assert output['mainParentHash'] == '""'
     assert output['sender'] == '""'
     assert output['shardId'] == '"rchain"'
-    assert output['bondsValidatorList'] == '02ab69930f74b931209df3ce54e3993674ab3e7c98f715608a5e74048b332821: 43,043c56051a613623cd024976427c073fe9c198ac2b98315a4baff9d333fbb42e: 15,1cd8bf79a2c1bd0afa160f6cdfeb8597257e48135c9bf5e4823f2875a1492c97: 2,23bb89653c1d43578ed421e655e7a0ed9f3ed2e7eab820ad7739277e380cafa3: 81'
+    assert output['bondsValidatorList'] == '02ab69930f74b931209df3ce54e3993674ab3e7c98f715608a5e74048b332821: 43#$043c56051a613623cd024976427c073fe9c198ac2b98315a4baff9d333fbb42e: 15#$1cd8bf79a2c1bd0afa160f6cdfeb8597257e48135c9bf5e4823f2875a1492c97: 2#$23bb89653c1d43578ed421e655e7a0ed9f3ed2e7eab820ad7739277e380cafa3: 81'
+    assert output['deployCost'] == r'User: , Cost: 132 DeployData #1553171134886 -- new x in { x!(0) }\\n}#$User: , Cost: 132 DeployData #1553171478932 -- new x in { x!(0) }\\n}'
 
 
 def test_extract_validator_stake_from_bonds_validator_str() -> None:
-    input = r'''02ab69930f74b931209df3ce54e3993674ab3e7c98f715608a5e74048b332821: 43,043c56051a613623cd024976427c073fe9c198ac2b98315a4baff9d333fbb42e: 15,1cd8bf79a2c1bd0afa160f6cdfeb8597257e48135c9bf5e4823f2875a1492c97: 2,23bb89653c1d43578ed421e655e7a0ed9f3ed2e7eab820ad7739277e380cafa3: 81'''
+    input = r'''02ab69930f74b931209df3ce54e3993674ab3e7c98f715608a5e74048b332821: 43#$043c56051a613623cd024976427c073fe9c198ac2b98315a4baff9d333fbb42e: 15#$1cd8bf79a2c1bd0afa160f6cdfeb8597257e48135c9bf5e4823f2875a1492c97: 2#$23bb89653c1d43578ed421e655e7a0ed9f3ed2e7eab820ad7739277e380cafa3: 81'''
```

Comment:
So #$ is the delimiter here? Where does it come from?

Reply:
https://github.com/rchain/rchain/blob/24c007f5cf7544a39215b243635f5025677ca182/integration-tests/test/rnode.py#L146-L147

Reply:
https://github.com/rchain/rchain/blob/24c007f5cf7544a39215b243635f5025677ca182/integration-tests/test/rnode.py#L41

Reply:
Got it